sympy/assumptions/refine.py

```diff
@@ -239,6 +240,16 @@ def refine_Relational(expr, assumptions):
     """
     return ask(Q.is_true(expr), assumptions)
 
+def refine_matrixelement(expr,assumptions):
+
+
+
+    x=expr.parent
+    if (ask(Q.symmetric(x),assumptions)==True):
+        if(expr.args[1]+expr.args[2]>2*expr.args[2]):
```

Comment:
As @asmeurer  said `(i - j).could_extract_minus_sign()` could better handle symbolic expressions.

Reply:
Could you elaborate this a little?

Reply:
The way I see, it returns True if `i>j` when `i` and `j` are both numbers (in SymPy). But in case of symbols it checks for the `canonical` form of the indices ( both being symbols) hence it's better to use this.
Also isn't the expression equivalent to `expr.args[1] > expr.args[2]` ?

Reply:
> Also isn't the expression equivalent to `expr.args[1] > expr.args[2]` ?

Yeah just noticed this. Will correct it.

Reply:
> `(i - j).could_extract_minus_sign()` I believe you should use it.